Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/inbound/PassthroughInboundDocRetrieve.java

```diff
@@ -43,32 +42,30 @@
 public class PassthroughInboundDocRetrieve extends BaseInboundDocRetrieve {
 
     private final PolicyTransformer pt;
-    private final AuditTransformer at;
     private final InboundDelegate ad;
     private final CONNECTInboundOrchestrator orch;
+    DocRetrieveAuditLogger docRetrieveAuditLogger = new DocRetrieveAuditLogger();
```

Comment:
I removed this - instance creation statement as this is no longer used as - getLogger() method gets a instance of audit logger.